source/run-commands.txt

```diff
@@ -0,0 +1,57 @@
+.. _run-commands:
+
+===========================
+Run Commands in ``mongosh``
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+For documentation of basic MongoDB operations in ``mongosh``, see:
+
+- :doc:`/crud`
+
+- :doc:`/run-agg-pipelines`
+
+- :doc:`/reference/methods`
+
+Terminate a Running Operation
+-----------------------------
+
+To terminate a running command or query in ``mongosh``,
+press ``Ctrl + C``.
+
+.. note::
+
+   ``Ctrl+C`` terminates the process in the shell, but does not
+   terminate the process on the MongoDB server.
+
+.. seealso::
+
+   :method:`db.killOp()`
```

Comment:
Perhaps it might make sense to specify that this does _not_ use killOp and that killOp can be used to kill commands that are running on the server?